src/css/profile/wearable/changeable/theme-circle/arclistview.less

```diff
@@ -4,7 +4,7 @@
 	overflow: hidden;
 	top: 0;
 	left: 0;
-
+	min-height: 250* @unit_base;
```

Comment:
OK with unit base, but after some additional check I think that the best place for definition is theme.layout.less file which defines similar items 
e.g.
https://github.com/Samsung/TAU/blob/tau_1.0/src/css/profile/wearable/changeable/theme-changeable/theme.layout.less
https://github.com/Samsung/TAU/blob/tau_1.0/src/css/profile/wearable/changeable/theme-circle/theme.layout.less

Reply:
I think it is consistent with code-style of the rest LESS files. 
**Example** 
https://github.com/Samsung/TAU/blob/tau_1.0/src/css/profile/wearable/changeable/theme-circle/listview.less#L37

Reply:
OK